capellambse/_namespaces.py

```diff
@@ -132,9 +112,9 @@ def get_keys_and_plugins_from_namespaces_by_url(
     ------
     UnsupportedPluginError
         If requested ``url`` is not found in :class:`NAMESPACES`.
-    ValueError
-        If a namespace key, plugin-name pair is found more than once
-        in :class:`NAMESPACES`.
+    AssertionError
+        If a plugin is registered multiple times
+        (with different namespace-keys) in :class:`NAMESPACES`.
```

Comment:
`assert` statements shouldn't be documented. In fact they mustn't, because they will get stripped by `python -O`.